@storyblok-components/video-component/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@storyblok-components/video-component",
+  "description": "Video Storyblok component",
+  "author": "Global Creative",
+  "license": "MIT",
+  "version": "0.1.0",
+  "babel": {
+    "presets": [
+      "@babel/preset-react"
+    ]
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.modern.js",
+  "scripts": {
+    "build": "microbundle --jsx React.createElement && find ./src -name \\*.sb.js -exec cp {} dist \\;"
+  },
+  "peerDependencies": {
+    "react": "^16.4.0",
+    "react-dom": "^16.8.6",
+    "react-scripts": "3.0.1"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "keywords": [
+    "@storyblok-components",
+    "storyblok",
+    "component",
+    "video"
+  ],
+  "devDependencies": {
+    "@babel/cli": "^7.5.5",
+    "@babel/core": "^7.8.4",
+    "@babel/preset-react": "^7.0.0",
+    "microbundle": "^0.11.0"
+  },
+  "publishConfig": {
+    "registry": "https://npmregistry.ef.com",
+    "access": "public"
+  },
```

Comment:
this is private repo, so you have to remove it

Reply:
it will not deploy

Reply:
oops